src/config

```diff
@@ -0,0 +1,13 @@
+{
+	".url": {
+		"SYNO.SDS.Tailscale": {
+			"type": "url",
+			"version": "1.4.5",
```

Comment:
can you use variables like ${VERSION} here from src/INFO.sh? Probably not? Should we auto-generate this file?

Reply:
We should generate it, leaving for a later pass after Maisem's code is in.